conans/client/command.py

```diff
@@ -1148,9 +1148,29 @@ def remote(self, *args):
                                             "a package recipe")
         parser_pupd.add_argument('reference', help='Package recipe reference')
         parser_pupd.add_argument('remote', help='Name of the remote')
+
+        list_pref = subparsers.add_parser('list_pref', help='List the package binaries and '
+                                                            'its associated remotes')
+        list_pref.add_argument('reference', help='Package recipe reference')
+
+        add_pref = subparsers.add_parser('add_pref',
+                                         help="Associate a package reference to a remote")
+        add_pref.add_argument('package_reference', help='Binary package reference')
+        add_pref.add_argument('remote', help='Name of the remote')
+
+        remove_pref = subparsers.add_parser('remove_pref', help="Dissociate a package's reference "
+                                                              "and its remote")
+        remove_pref.add_argument('package_reference', help='Binary package reference')
+
+        update_pref = subparsers.add_parser('update_pref', help="Update the remote associated with "
+                                            "a binary package")
+        update_pref.add_argument('package_reference', help='Bianary package reference')
+        update_pref.add_argument('remote', help='Name of the remote')
```

Comment:
I wonder why the user would need to edit the ``registry`` file 😕

Reply:
Letting users do whatever they want. However, with the new usage of remote info planned for conan 2.0, it could be that the commands to edit the registry are not necessary anymore.

Reply:
I think they still will need to change some references, see my previous comment about the upload not updating the references: 

> This is an internal decision (probably arbitrary), that if you are installing from R1 but 
  uploading to R2, you want the updates still from R1, otherwise you need to change the 
  reference manually.

You always will need to be able to change the references no mater where they are, right?